PharoLauncherTutorial/pharolauncher_tutorial.pillar

```diff
@@ -0,0 +1,130 @@
+
+! PharoLauncher Tutorial: What ? How ?
+
+
+This unit aims to provide a basic overview of what PharoLauncher is and give a step-by-step approach on how to use this application of great advantage. 
+
+
+!!Overview
+
+Pharo is an open source implementation of the programming language and environment Smalltalk. Pharo is not Smalltalk. Pharo is Smalltalk-inspired. 
```

Comment:
This section is really nice, but doesn't fit well here as there presumably will be another chapter giving an overview of Pharo as a system. Save this and we'll see if there's somewhere else we can include it.